app/components/chat-message.js

```diff
@@ -5,6 +5,7 @@ import { action } from '@ember/object';
 export default class ChatMessage extends Component {
   @tracked gifsEnabled = true;
   @tracked imgRegex = /https?:\/\/(?:[a-z0-9-]+\.)+[a-z]{2,6}(?:\/[^/#?]+)+\.(?:jpg|gif|png|webp)(\?.*$)*/;
+  @tracked dataRegex = /data:image\/(?:png|jpeg|webp);base64,.+/;
```

Comment:
These regex's  don't need to be tracked.

Reply:
yeah i was wondering about that